tests/lib/SimpleSAML/ConfigurationTest.php

```diff
@@ -31,6 +31,13 @@ public function testLoadDefaultInstance()
      */
     public function testCriticalConfigurationError()
     {
+        try {
+            SimpleSAML_Configuration::getInstance();
+            $this->fail('Exception expected');
+        } catch (\SimpleSAML\Error\CriticalConfigurationError $var) {
+            // This exception is expected.
```

Comment:
This is normal behaviour. `SimpleSAML_Configuration::getInstance()` attempts to fetch the configuration from disk. If there's no such configuration, it fails with a `CriticalConfigurationError` exception.